index.html

```diff
@@ -159,12 +172,12 @@ <h2 class="card-title">Multi-Post Stories</h2>
                 </div>
               </div>
               <div class="card-footer">
-                <button class="btn">See Project</button>
+                <button class="btn pop-toggler">See Project</button>
               </div>
             </div>
           </article>
           <div class="project-cards-container">
-            <article class="project-card-variant printing">
+            <article class="project-card-variant printing" data-name="printing">
```

Comment:
- [ ] Kindly remove the projects cards from the HTML and add them dynamically using JavaScript as mentioned in the project requirements 👇 

![image](https://user-images.githubusercontent.com/50721479/208717086-55194c74-d146-4279-a986-021e5e39c1ba.png)